gapic/api/artman_bigtable.yaml

```diff
@@ -1,6 +1,7 @@
 common:
   api_name: bigtable
   api_version: v2
+  package_type: grpc_client
```

Comment:
Given that the vast majority of artman configs are likely to be gRPC clients, can this just be a default setting, so that the field only needs to be configured if it's *not* a gRPC client?